test/e2e-v2/java-test-service/Dockerfile.consumer

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG SW_AGENT_JDK_VERSION=8
+ARG SW_AGENT_JAVA_COMMIT=1f3c08a5af19f8522f2a40d9339c45fa816bfe07
```

Comment:
Miss change?

Reply:
Could we remove the default value and use GHA env variable only?

Reply:
Bear in mind the tests will be run locally, only depending on GHA env var  requires requires extra settings when running locally

Reply:
It helps for e2e testing locally, so I think we should not remove it.

Reply:
OK, then, I just feel this will be forgotten to update with time.